magda-storage-api/src/createApiRouter.ts

```diff
@@ -98,7 +134,29 @@ export default function createApiRouter(options: ApiRouterOptions) {
             });
     });
 
-    // Remove/Delete an object
+    /**
+     * @apiGroup Storage API
+     *
+     * @api {delete} /v0/{bucket}/{fieldid} Request to delete an object at {bucket} with name {fieldid}
+     *
+     * @apiDescription Deletes an object. This is a hard delete, and cannot be undone.
+     * Note that if the {fieldid} does not exist, the request will not fail.
+     *
+     * @apiParam (Request body) {string} bucket The name of the bucket where the object resides
+     * @apiParam (Request body) {string} fieldid The name of the object to be deleted
+     *
+     * @apiSuccessExample {json} 200
+     *    {
+     *        "message":"File deleted successfully",
+     *    }
+     *
+     * @apiError {string} Internal Server Error
```

Comment:
You'll need to take out these too, they result in weirdness like this:

<img width="1035" alt="image" src="https://user-images.githubusercontent.com/900555/72481855-df691880-384f-11ea-9796-9b412d7c68bc.png">